cockatrice/src/player.cpp

```diff
@@ -1136,6 +1136,9 @@ void Player::actCreateRelatedCard()
     // get the target card name
     QAction *action = static_cast<QAction *>(sender());
     CardInfo *cardInfo = db->getCard(action->text());
+    
+    // get the target token's location
+    QPoint gridPoint = QPoint(-1, table->clampValidTableRow(2 - cardInfo->getTableRow()));
```

Comment:
This seems worth wrapping in a function with a comment explaining what this is doing, and why `-1` and `2` are used.